src/NLU.DevOps.Luis/ILuisClient.cs

```diff
@@ -4,87 +4,11 @@
 namespace NLU.DevOps.Luis
 {
     using System;
-    using System.Collections.Generic;
-    using System.Threading;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.Models;
-    using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime.Models;
-    using Newtonsoft.Json.Linq;
 
     /// <summary>
     /// Interface for LUIS operations.
     /// </summary>
-    public interface ILuisClient : IDisposable
+    public interface ILuisClient : ILuisTrainingClient, ILuisPredictionClient, IDisposable
```

Comment:
What about `LuisAuthoringClient` and `LuisRuntimeClient` for more consistency with the LUIS nomenclature?

Reply:
I wanted it to be different because the `ILUISAuthoringClient` and `ILUISRuntimeClient` already exist in the LUIS SDK to avoid confusion between the two, but can switch back

Reply:
I wanted it to be different because the `ILUISAuthoringClient` and `ILUISRuntimeClient` already exist in the LUIS SDK to avoid confusion between the two, but can switch back